modules/login/php/passwordexpiry.class.inc

```diff
@@ -1,158 +0,0 @@
-<?php
```

Comment:
is this correct? to delete the 3 php classes?

Reply:
the module no longer uses them.